docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,607 @@
+  - challenge transaction is reverted.
+
+| Group creation - DKG acceptance
+| Anyone can request unlocking of the sortition pool and marking of the DKG
+  result as accepted (but not all requests will be processed positively).
+
+| Group creation - DKG acceptance
+| Upon processing the transaction unlocking the sortition pool and marking the
+  DKG result as accepted, the random beacon contract should determine validity
+  of the transaction.
+
+| Group creation - DKG acceptance
+| When determining validity of the tx unlocking the sortition pool and accepting
+  the DKG result, the following aspects should be taken into account: +
+  - is the sortition pool locked? +
+  - has the challenge period already passed? +
+  - is the sender eligible to accept the DKG result? +
+  Only if all the above conditions have been met, the tx is considered valid.
+
+| Group creation - DKG acceptance
+| At the beginning (right after DKG result is submitted on chain) only the result
+  submitter should be eligible to accept the DKG result and earn the reward.
+  After a fixed amount of time everybody should become eliglible to accept the
+  DKG result.
+
+| Group creation - DKG acceptance
+| If the transaction unlocking the sortition pool and marking the DKG result as
+  accepted gets approved, then: +
+  - the sortition pool should get unlocked +
+  - the DKG result should be accepted and a group should be created based on the
+  candidate group +
+  - the DKG result accepter should receive the fixed reward (in T), paid from
+  the rewards pool to the submitter’s address (in the same tx) +
+  - operators marked as inactive/disqualified during DKG protocol execution
+  should be punished by being excluded from earning rewards for a fixed amount
+  of time (in the same tx) +
+  - operators who became eligible for submitting the DKG result before the
+  member who submitted the DKG result should be punished by being excluded from
+  earning rewards for a fixed amount of time (in the same tx).
+
+| Group creation - DKG timeout
+| The DKG submission timeout equals the group size multiplied by the number of
+  blocks for a member to become eligible to submit the DKG result plus the
+  number of blocks covering for the time of DKG generation. The timer should
+  start when the first member becomes eligible (the moment when DKG was
+  requested). The timer gets reset when a valid DKG result challenge is
+  submitted. 
+
+| Group creation - DKG timeout
+| Anyone can send a transaction reporting DKG timeout.
+
+| Group creation - DKG timeout
+| Upon noticing the DKG timeout notification, the random beacon contract should
+  determine its validity.
+
+| Group creation - DKG timeout
+| When determining validity of the DKG timeout notification, the following
+  aspects should be taken into account: +
+  - does it reference the existing DKG request? (if not, then notification
+  invalid) +
+  - has the DKG timeout passed? (if not, then notification invalid) +
```

Comment:
Not sure if that condition need to be always passing. 

From the Random beacon README:
> DKG timeout includes the situation when no new relay entry was produced and sortition could not be performed.

That means this situation:
Somebody requested a new entry, frequency of group creation was hit, and due to that the sortition pool got locked. Then nobody published the new relay entry and we now want to unlock the pool and will do that in the DGK timeout notification.

From RFC-1 we know that:
> The [DKG timeout] timer starts at the moment when the first member becomes eligible [to submit DKG result]

My understanding is that in the case I described we never start the DKG, so we can't say whether the DKG timeout has passed or not...